web/src/pages/Route/List.tsx

```diff
@@ -542,6 +545,27 @@ const Page: React.FC = () => {
         actionRef={ref}
         rowKey="id"
         columns={columns}
+        rowSelection={rowSelection}
+        tableAlertRender={() => (
+          <Space size={24}>
+            <span>
+            {formatMessage({ id: 'page.route.chosen' })} {selectedRowKeys.length} {formatMessage({ id: 'page.route.item' })}
+            </span>
+          </Space>
+        )}
+        tableAlertOptionRender={() => {
+          return (
+            <Space size={16}>
+              <Button
```

Comment:
A small suggestion, for the delete type of button we tend to use red to keep the style consistent, for this delete all the dangerous operations, it is best to add a pop-up window to confirm the second

Reply:
![image](https://user-images.githubusercontent.com/72308243/181669888-06518971-cea0-4075-938a-f7369207b54d.png)
In fact, in Antd UI, deletion generally prefers White，when I have spare power, I will try to add a secondary message, and then open a small patch PR

Reply:
Please create an issue to track this issue